package.json

```diff
@@ -55,8 +55,8 @@
         "@babel/parser": "^7.17.0",
         "@badeball/cypress-cucumber-preprocessor": "^20.0.2",
         "@cypress/webpack-preprocessor": "^6.0.1",
-        "@dhis2/app-runtime": "^3.9.0",
-        "@dhis2/cli-app-scripts": "^11.7.0",
+        "@dhis2/app-runtime": "^3.11.2",
+        "@dhis2/cli-app-scripts": "^12.0.0-alpha.19",
```

Comment:
not sure how to manage this - probably should keep it as the version in master, until it's merged and bump it to v12